docs/docs/contribute/software/add-new-metric-provider.md

```diff
@@ -0,0 +1,123 @@
+# Adding a New Metrics Provider for Placeholder Endpoint
```

Comment:
```suggestion
---
comments: true
---

# Adding a New Metrics Provider for Placeholder Endpoint
```
This is required so readers can leave comments for the page.

Reply:
Alright